webapp/channels/src/components/widgets/settings/text_setting.tsx

```diff
@@ -5,7 +5,7 @@ import React from 'react';
 
 import Setting from './setting';
 
-export type InputTypes = 'input' | 'textarea' | 'number' | 'email' | 'tel' | 'url' | 'password'
+export type InputTypes = 'input' | 'textarea' | 'number' | 'email' | 'tel' | 'url' | 'password' | undefined
```

Comment:
do you need to pass explicitly undefined here?

Reply:
Since the type prop is optional if it isn't passed it will be undefined

Reply:
we dont need to do this as per https://github.com/mattermost/mattermost/pull/23821#discussion_r1246011918